microsoft-edge/webview2/how-to/print.md

```diff
@@ -0,0 +1,755 @@
+* [ICoreWebView2PrintSettings interface](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## The PrintToPdfStream method to customize printing to a PDF stream
+
+Silently prints the current top-level document in the WebView2 control to a PDF stream. To completely control how printing is performed, you can print to a PDF and then build your own code to print the PDF. This API consists of an asynchronous `PrintToPdfStream` method and a `PrintSettings` object.
+
+
+<!-- ------------------------------ -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.PrintToPdfStreamAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfstreamasync#microsoft-web-webview2-core-corewebview2-printtopdfstreamasync)
+* [CoreWebView2Environment.CreatePrintSettings Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createprintsettings)
+* [CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+
+<!-- ------------------------------ -->
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.PrintToPdfStreamAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#PrintToPdfStreamAsync)
+* [CoreWebView2Environment.CreatePrintSettings Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createprintsettings)
+* [CoreWebView2PrintSettings Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2printsettings)
+
+
+<!-- ------------------------------ -->
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_16::PrintToPdfStream method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#printtopdfstream)
+* [ICoreWebView2Environment6::CreatePrintSettings method](/microsoft-edge/webview2/reference/win32/icorewebview2environment6#createprintsettings)
+* [ICoreWebView2PrintSettings2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2printsettings2)
+
+---
+
+<!-- end of tab-set -->
+
+#### Example: The PrintToPdfStream method to customize printing to a PDF stream
+
+This example prints the PDF data of the current webpage to a stream.
+
+
+<!-- ------------------------------ -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+```csharp
+async void PrintToPdfStream()
+{
+  try
+  {
+    string title = webView.CoreWebView2.DocumentTitle;
+
+    // Passing null for `PrintSettings` results in default print settings used.
+    System.IO.Stream stream = await webView.CoreWebView2.PrintToPdfStreamAsync(null);
+    DisplayPdfDataInPrintDialog(stream);
+
+    MessageBox.Show(this, "Printing " + title + " document to PDF Stream " + ((stream != null) ? "succedded" : "failed"), "Print To PDF Stream");
```

Comment:
GLOBAL: Nice to do per Docs Contributor Guide: wrap at or near column 85.  
Check how unreadable the code is in the narrow review.dmc rendering; see how often a code block has horizontal scrollbar.

Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/print?branch=pr-en-us-2403#example-the-printtopdfstream-method-to-customize-printing-to-a-pdf-stream

This line is the only spot in the article that has the (undesirable) horizontal scrollbar according to the wide, GH-rendered view:
https://github.com/MicrosoftDocs/edge-developer/blob/user/monicach/PrintConceptDoc/microsoft-edge/webview2/how-to/print.md#example-the-printtopdfstream-method-to-customize-printing-to-a-pdf-stream